### PR TITLE
1689 Ignore HTTP 400 error messages sent by the general-info API

### PR DESCRIPTION
**Pull request by herbepau**
...and 1406 Hard coded secret word indexes during new wallet creation

@carlton355 @fenix2222

### CODE REVIEW

File: Breeze.UI/src/app/wallet/tumblebit/tumblebit.component.ts [hunk 1]
```diff
@@ -217,10 +217,11 @@ export class TumblebitComponent implements OnDestroy {
             this.genericModalService.openModal(
               Error.toDialogOptions('Failed to get general wallet information. Reason: API is not responding or timing out.', null));
           } else if (error.status >= 400) {
-            if (!error.json().errors[0]) {
+            const firstError = error.json().errors[0];
```
Comment (fenix2222) [inline Breeze.UI/src/app/wallet/tumblebit/tumblebit.component.ts]: You could use getFirstError from Error helper class https://github.com/BreezeHub/BreezeProject/blob/master/Breeze.UI/src/app/shared/classes/error.ts

File: Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export class SecretWordIndexGenerator {
+    
+    private readonly textPrefix = 'Word °';
+    readonly index1 = SecretWordIndexGenerator.getRandom();
+    readonly index2 = SecretWordIndexGenerator.getRandom(this.index1);
+    readonly index3 = SecretWordIndexGenerator.getRandom(this.index1, this.index2);
+    readonly text1 = `${this.textPrefix}${this.index1 +1 }`;
```
Comment (fenix2222) [inline Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts]: Do these have to be in order? Like 2, 7, 9 and not 5, 1, 4?

File: Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+export class SecretWordIndexGenerator {
+    
+    private readonly textPrefix = 'Word °';
```
Comment (fenix2222) [inline Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts]: Prefix used to be Word n°, not Word °

Comment (herbepau) [reply Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts]: good spot, ok

File: Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts (hunk 2 above)
Comment (herbepau) [reply Breeze.UI/src/app/setup/create/confirm-mnemonic/secret-word-index-generator.ts]: I understand following chat

**Pull request merged**